Remove themes

rickvdbosch opened the pull request:
Closes #122

jfversluis commented:
Hm OK, wasn't expecting this :D 
I'm not sure if it's that we can just simply remove it and be done with it. I think there is still some npm building stuff going on? This still works on your end?

Also, funny how the testing seems to work on this PR...

rickvdbosch commented:
I checked and didn't see any (other) building stuff going on. There doesn't seem to be any build events or scripts. There also doesn't seem to be any references to anything in the Themes folder from the projects in the solution. I removed the `npm` stuff from the build pipeline as well. 

As far as the testing goes: I changed some stuff on the package references for the test projects to see if it fixes the testing issues. Looks good 😁
